packages/pwa/app/pages/cart/index.jsx

```diff
@@ -67,7 +67,8 @@ const Cart = () => {
                 title: formatMessage(
                     {
                         defaultMessage:
-                            '{quantity} {quantity, plural, one {item} other {items}} added to wishlist'
+                            '{quantity} {quantity, plural, one {item} other {items}} added to wishlist',
+                        id: 'cart.info.added_to_wishlist'
```

Comment:
Since we do have some cart components, it is clearer to say `cart_page` for the ids here
```suggestion
                        id: 'cart_page.info.added_to_wishlist'
```

Reply:
🤔 Since we don't have an actual `<Cart>` component, I think this is ok as it is.